### PR TITLE
Feature/gv game perc2

### PR DESCRIPTION
**Pull request by GVarenhorst**
## High level what you did: 
Created and passed percentage_home_wins, percentage_visitor_wins, and percentage_ties methods
### Details:
- Created methods in game_teams file, then moved to game_statistics.rb
- Created tests in game_statistics_spec.rb 
## Questions:
None
## Overall:
- [x] Tests are written for all methods added
- [x] Rspec running clean
- [ ] Bugs, need help, etc.

### CODE REVIEW

File: spec/game_teams_spec.rb [hunk 1]
```diff
@@ -58,4 +57,10 @@
       expect(@stat_tracker.game_teams).to all(be_a(GameTeams))
     end
   end
+
+  describe "#percentage home" do
+    it "can calculate percentage for home wins" do
+      expect(@stat_tracker.game_teams.percentage_home_wins).to eq(0.600)
+    end
+  end
```
Comment (aplasket) [inline spec/game_teams_spec.rb]: @GVarenhorst can we delete out ln 61 -65 here?

File: spec/game_statistics_spec.rb [hunk 2]
```diff
@@ -45,7 +45,19 @@
 
   describe "#percentage_home_wins" do
     it "returns a percentage of games that a home team has won as a float (rounded to the nearest 100th)" do
-      # expect(@game_stats.percentage_home_wins).to be_a(Float)
+      expect(@game_stats.percentage_home_wins).to be_a(Float)
+    end
```
Comment (aplasket) [inline spec/game_statistics_spec.rb:50]: we should also make the assertion:
`expect(@game_stats.percentage_home_wins).to eq(0.44)`

File: spec/game_statistics_spec.rb [hunk 3]
```diff
@@ -45,7 +45,19 @@
...
+  end
+
+  describe "#percentage_visitor_wins" do
+    it "returns a percentage of games that a visiting team has won as a float (rounded to the nearest 100th)" do
+      expect(@game_stats.percentage_visitor_wins).to be_a(Float)
```
Comment (aplasket) [inline spec/game_statistics_spec.rb:55]: we should also make the assertion:
expect(@game_stats.percentage_visitor_wins).to eq(0.36)

File: spec/game_statistics_spec.rb [hunk 4]
```diff
@@ -45,7 +45,19 @@
...
+    end
+  end
+
+    describe "#percentage_ties" do
+      it "returns a percentage of games that both teams have won as a float (rounded to the nearest 100th)" do
+        expect(@game_stats.percentage_home_wins).to be_a(Float)
```
Comment (aplasket) [inline spec/game_statistics_spec.rb]: we should also make an assertion here that:
 expect(@game_stats.percentage_ties).to eq(0.20)

**Comment by goldenll** [comment]
Looks good to me after AP's suggested changes are made

File: spec/game_statistics_spec.rb (hunk 2 above)
Comment (GVarenhorst) [reply spec/game_statistics_spec.rb:50]: Updated

File: spec/game_statistics_spec.rb (hunk 3 above)
Comment (GVarenhorst) [reply spec/game_statistics_spec.rb:55]: Updated

File: spec/game_statistics_spec.rb (hunk 4 above)
Comment (GVarenhorst) [reply spec/game_statistics_spec.rb]: Updated and corrected line 60 expect statement

**Pull request merged**